proto/kvrpcpb.proto

```diff
@@ -562,3 +562,11 @@ message ReadIndexResponse{
     uint64 read_index = 2;
 }
 
+message GetAppliedIndexRequest {
+    uint64 region_id = 1;
```

Comment:
How about add a new service `MultiDCProxy`, and put a new RPC `GetCommittedIndexAndTs` in it? BTW we need to contains a `Context` in the request.